infra/base-images/base-runner/bad_build_check

```diff
@@ -434,7 +440,12 @@ function main {
   result=$?
   checks_failed=$(( $checks_failed + $result ))
 
-  check_mixed_sanitizers $FUZZER
+  if [[ "$FUZZING_ENGINE" == centipede \
+        && "$SANITIZER" != none && "${HELPER:-}" == True ]]; then
```

Comment:
What does `"${HELPER:-}"` do?

Reply:
It [substitutes `HELPER` with an empty string](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html) and avoids the '[unbounded variable error](https://bnikolic.co.uk/blog/bash-unbound-variable)' when `HELPER` is not set (e.g., when not using `helper.py`).
I choose this way because it is consistent with [other places](https://github.com/google/oss-fuzz/blob/86e6dc3e2ab6665711931f0c53bff8e0181d7114/infra/base-images/base-runner/bad_build_check#L312).